applications/GeoMechanicsApplication/custom_geometries/line_interface_geometry.h

```diff
@@ -208,6 +208,26 @@ class LineInterfaceGeometry : public Geometry<Node>
         KRATOS_ERROR << IntegrationSchemeFunctionalityNotImplementedMessage();
     }
 
+    GeometriesArrayType GenerateEdges() const override
+    {
+        const auto points = this->Points();
+
+        // The first edge coincides with the first side of the element
+        auto begin_of_second_side = points.ptr_begin() + (points.size() / 2);
+        const auto nodes_of_first_edge = PointerVector<Node>{points.ptr_begin(), begin_of_second_side};
+
+        // The second edge coincides with the second side of the element. However, the nodes must be
+        // traversed in opposite direction.
+        auto nodes_of_second_edge = PointerVector<Node>{begin_of_second_side, points.ptr_end()};
+        std::swap(nodes_of_second_edge(0), nodes_of_second_edge(1)); // end nodes
+        std::reverse(nodes_of_second_edge.ptr_begin() + 2, nodes_of_second_edge.ptr_end()); // any high-order nodes
```

Comment:
Nice, should indeed work generically. But in the tests have 0 or 1 midpoints so I doubt if this really is tested.

Reply:
You are right, the actual reversal of the high-order nodes is _not_ covered by the unit tests. In fact, the current tests would still pass if we were to take out the reversal of the high-order nodes. The reason I have added it, is to make the implementation complete and understandable. Otherwise, we might get surprising results when we start to make 4+4 or 5+5 line interface elements available. What are your thoughts on this?

Reply:
Leave as is for now, We won't make higher order interface elements without unit tests, so this will be handled in future.